kurzushetvege/gyenes_vizu.py

```diff
@@ -0,0 +1,35 @@
+import pandas as pd
+import numpy as np
+import itertools
+import networkx as nx
+from pyvis import network as net
+from IPython.core.display import display, HTML
+
```

Comment:
ezt csomagold be egy függvénybe, ami legyen az kb h generate_dummy_data(), és adja vissza node_features, edge_features változókat